Closes #1876​

nivaca opened the pull request:
This PR closes #1876​ . It is a maintenance for the *Transforming Data for Reuse and Re-publication with XML and XSL* lesson, both in EN and ES.  

### Checklist

- [X] Assign yourself in the "Assignees" menu
- [X] Assign at least one individual or team to "Reviewers"
- [ ] if the text needs to be translated, assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing edtor in your PR. Please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines) when writing your PR description
- [X] Add the appropriate "Label"
- [X] [Ensure the status checks pass](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#4-check-your-pr-status)
- [X] [Check the live preview of your PR on Netlify](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#5-preview-how-your-pr-looks-when-built-into-html)
- [X] If this PR closes an open issue, add the phrase `Closes #ISSUENUMBER` to the description above

*If you are having difficulty fixing build errors, first consult <https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions> carefully, especially ["Common Build Errors"](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#common-build-errors). Then contact the technical team if you need further help.*

nivaca committed:
updated images for ES and ES versions of transforming-xml-with-xsl

nivaca committed:
updated ES and ES versions of transforming-xml-with-xsl

nivaca committed:
added required assets for the ES and ES versions of transforming-xml-with-xsl

nivaca committed:
updated the ES and ES versions of transforming-xml-with-xsl to include correct assets information

anisa-hawes commented:
Hello @nivaca.

Thank you so much for all your work on this.

May I make a couple of small adjustments to your forked branch here? https://github.com/nivaca/jekyll/tree/gh-pages/en/lessons

at Line 22: <div class="alert alert-warning"> I think this could be an "information" box rather than a "warning" now <div class="alert alert-info”> 

at Line 27:  insert {% include toc.html %} to display the Table of Contents before #Introduction

Throughout: I note that some of the code blocks don’t appear to be rendering in the netlify Lesson Preview. e.g, Lines 49-55, 59066, and 71-79. I think the backicks [```] may need to be on the line above the first line of code?

P.S. I have edited your comment above, removing the sentence "(Note that PT and FR translations should also be updated.)" In this case, the lesson is only available in EN and ES at the moment!

anisa-hawes committed:
Update transforming-xml-with-xsl.md

Line 22: <div class="alert alert-warning"> -> <div class="alert alert-info”> 

Line 27:  insert {% include toc.html %} to display the Table of Contents before #Introduction

Lines 49-55, 59-66, and 71-79. The code blocks don’t appear to be rendering in our netlify Lesson Preview. I am moving the backicks [```] so that they are on the line above the first line of code.

anisa-hawes committed:
Update transformacion-datos-xml-xsl.md

Line 22: <div class="alert alert-warning"> -> <div class="alert alert-info”>

nivaca commented:
Hello @anisa-hawes,
Great! Many thanks for doing so!

anisa-hawes committed:
Update transforming-xml-with-xsl.md

Adjusting backticks [```] above code blocks throughout.

anisa-hawes committed:
Update transforming-xml-with-xsl.md

Replacing uppercase XML with lowercase xml in syntax highlighting.

anisa-hawes committed:
Update transformacion-datos-xml-xsl.md

Replacing uppercase XML with lowercase xml in syntax highlighting.

rivaquiroga commented:
@anisa-hawes, there is an issue in the way the notes are rendering in the netlify preview. I can work on fixing it tomorrow if needed.

anisa-hawes commented:
Thank you, @rivaquiroga! Yes, @nivaca and I have been exchanging emails about this too...

We note that `<raíz>` looks odd. We may need to escape the special character `í`?

Also, in ES the directory path `C:\Users\usuario\Downloads\SaxonHE11-2J>` is displaying strangely. 

Your help will be much appreciated. I am around tomorrow, so we can work together.

anisa-hawes committed:
Update transforming-xml-with-xsl.md

Insert missing line of code (line 218).

anisa-hawes commented:
Hello @rivaquiroga,

Let me know if you have time to take a look at this today or tomorrow? I'm around, so we can work together. 

Thank you.

rivaquiroga committed:
fix EN footnotes indenting

rivaquiroga committed:
I forgot to indent one line

rivaquiroga committed:
fix ES footnotes indenting

rivaquiroga committed:
fix broken link

rivaquiroga committed:
remove h2 headers in footnotes

anisa-hawes commented:
Thank you so much, @rivaquiroga! ✨

rivaquiroga committed:
some copyediting in ES translation

rivaquiroga commented:
I fixed the indentation of the footnotes in both English and Spanish. Line breaks inside a footnote need to be indented for them to be rendered properly.
Regarding the two sets of footnotes in the Spanish lesson, it is not possible to have them because footnotes are numbered sequentially in markdown, so you can't separte them in two different sections. I put a "Nota del traductor:" at the beginning of the ones that the translator added. 

@anisa-hawes, I changed [this link](https://chnm.gmu.edu/essays-on-history-new-media/essays/?essayid=6) in [this blog post](https://programminghistorian.org/posts/distant-reading-in-the-undergraduate-classroom) because the site is on maintenance. It looks that the archived version doesn't show the content as expected, so it would be a good idea to check in couple of days is the site is up again.

anisa-hawes commented:
Enormous thanks, @rivaquiroga! Yes, I'll set myself a reminder to recheck the link you've adjusted in a few days' time. 

Hello @svmelton. This is the Lesson Maintenance task Nicolás Vaughan has been working on. In this case, new DOIs will be needed (one for EN, one for ES).

rivaquiroga commented:
I'll request the new DOI for the ES translation!

anisa-hawes committed:
Update transforming-xml-with-xsl.md

- para.47 sub-sections
- lines 129-131 removed punctuation and conjunctions from list items
- line 195 format link
- line 203 format link
- para.207 may be named differently in your case
- line 601 each record on its own line
- para.603 If you wish
- lines 774-780 removed format information from bibliographic references

anisa-hawes commented:
(I've just contributed some minor copyedits for EN)
